framework/filters/Cors.php

```diff
@@ -80,6 +80,10 @@ class Cors extends ActionFilter
      */
     public $actions = [];
     /**
+     * @var bool handle the preflight check and don't proceed with the execution of the request.
+     */
+    public $handleOptions = true;
```

Comment:
Thought about it more. We should not introduce this property. Instead, we may handle preflight request specifically:

```php
if ($this->request->isOptions && $this->request->headers->has('Access-Control-Request-Method')) {
    // it is CORS preflight request, respond with 200 OK without further processing
    $this->response->setStatusCode(200);
    return false;
}
```